scripts/init/Initialize-DownloadLatest.ps1

```diff
@@ -5,6 +5,8 @@
     [Parameter(Mandatory=$false)] [boolean] $unzip = $false
 )
 
+$ErrorActionPreference = "Stop"
```

Comment:
very minor nit but we might want to include this on Initialize-InstallFromNuGet.ps1 as well.